runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineRunner.java

```diff
@@ -212,6 +246,22 @@ public PortablePipelineResult run(RunnerApi.Pipeline pipeline, JobInfo jobInfo)
             pipelineOptions.as(MetricsOptions.class),
             result);
     metricsPusher.start();
+    if (pipelineOptions.getEventLogEnabled()) {
+      eventLoggingListener.onApplicationStart(
+          new SparkListenerApplicationStart(
+              jobInfo.jobId(),
+              scala.Option.apply(jobInfo.jobName()),
+              Instant.now().getMillis(),
```

Comment:
This block runs after the job has completed, so this would report the start time being almost exactly the same as the end time.

Reply:
Ok, now I'm using startTime.